apps/api/src/domains/project/issue/issue.service.ts

```diff
@@ -177,10 +172,7 @@ export class IssueService {
     ) {
       throw new IssueInvalidNameException('Duplicated name');
     }
-    await this.repository.update(
-      { id: issueId },
-      { id: issueId, name, description, status, externalIssueId },
-    );
+    await this.repository.save(Object.assign(issue, dto));
```

Comment:
@chiol 
We should talk about this situation.
I just assigned `dto`to the `issue` that was already saved in the database.
However if you see the elements of the dto, there is `issueId` which is not using in `IssueEntity`.
I just ignored it because it does not occur any errors... but should we change the code not to assign just dto to `issue` but to assign using elements?

```ts
const { issueId, name, ...rest } = dto;
await this.repository.save(Object.assign(issue, { name, ...rest }));
```